aiida_pseudo/groups/family/sssp.py

```diff
@@ -50,15 +50,19 @@ def format_configuration_label(cls, configuration: SsspConfiguration) -> str:
         )
 
     @classmethod
-    def format_configuration_filename(cls, configuration: SsspConfiguration, extension: str) -> str:
+    def format_configuration_filename(
+        cls, configuration: SsspConfiguration, extension: str, patch_version: Optional[str] = None
```

Comment:
Please update the docstring to explain what the `patch_version` looks like and what its purpose is.

Reply:
![aye-aye-captain](https://user-images.githubusercontent.com/25607960/129154890-cc2ba733-77e6-4bec-a9f2-5722b802e224.gif)